genshin/utility/logfile.py

```diff
@@ -13,40 +13,52 @@
 AUTHKEY_FILE = fs.get_tempdir() / "genshin_authkey.txt"
 
 
-def get_logfile() -> typing.Optional[pathlib.Path]:
+def get_logfile(game_location: typing.Optional[PathLike] = None) -> typing.Optional[pathlib.Path]:
```

Comment:
not necessarily that important but maybe name it `cache_location`/`cachefile` or something?